Core/PSPLoaders.cpp

```diff
@@ -322,5 +323,41 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 	pspFileSystem.Mount("umd0:", fs);
 
 	std::string finalName = ms_path + file + extension;
+
+	std::string homebrewName = PSP_CoreParameter().fileToStart;
+	std::size_t lslash = homebrewName.find_last_of("/");
+	homebrewName = homebrewName.substr(lslash +1);
+	std::string madeUpID = g_paramSFO.GenerateFakeID();
+
+	char title[1024];
+	sprintf(title, "%s : %s", madeUpID.c_str(), homebrewName.c_str());
```

Comment:
Not sure how to do the other, so StringFromFormat there is.:3 BTW that line was mostly a copy paste of code used in full games/demos already. Should I change that as well?